pkg/controller/directvolumemigration/task.go

```diff
@@ -89,33 +91,33 @@ func (r Itinerary) progressReport(phase string) (string, int, int) {
 var VolumeMigration = Itinerary{
 	Name: "VolumeMigration",
 	Steps: []Step{
-		{phase: Created},
-		{phase: Started},
-		{phase: Prepare},
-		{phase: CreateDestinationNamespaces},
-		{phase: DestinationNamespacesCreated},
-		{phase: CreateDestinationPVCs},
-		{phase: DestinationPVCsCreated},
-		{phase: CreateRsyncRoute},
-		{phase: CreateRsyncConfig},
-		{phase: CreateStunnelConfig},
-		{phase: CreatePVProgressCRs},
-		{phase: CreateRsyncTransferPods},
-		{phase: WaitForRsyncTransferPodsRunning},
-		{phase: CreateStunnelClientPods},
-		{phase: WaitForStunnelClientPodsRunning},
-		{phase: CreateRsyncClientPods},
-		{phase: WaitForRsyncClientPodsCompleted},
-		{phase: DeleteRsyncResources},
-		{phase: Completed},
+		{phase: Created, phaseDescription: "The DVM controller just saw DVM CR being created"},
```

Comment:
I would prefer separating the phase descriptions from the itinerary so that this is in line with how phase descriptions are implemented in the migmigration controller.